README.md

```diff
@@ -78,3 +78,10 @@ The most common environment variables to set would include:
 - If a server fails, it is auto-detected and the work of the server is redistributed to the remaining servers in the cluster
 - If a server joins, work is redistributed to include the new server
 
+## Development
+
+### Git hooks
+
+- Run `bb git-hooks install`
```

Comment:
This is what I'd need to do for this to work going forwards, right?

Reply:
Yep, or perhaps more properly `bb run git-hooks install` but this should work too.